README.md

```diff
@@ -282,3 +282,11 @@ Deleted posts are restored automatically before the push occurs. However, you ca
 ```bash
 npm run dev-restore
 ```
+
+### Optimizing Images
+
+The pre-push workflow will validate image size and fail if there are images bigger than 400kb in size. It will also warn you about using PNG images as they are lossless and occupy more space. For a blog post, JPEGs or WebP images are better suited. So do consider using those formats. And ideally, images should be under 1800px in width The blog content is rendered in a 900px width container and [retina images](https://www.sleeplessmedia.com/2018/12/14/optimizing-website-images-and-graphics-for-retina-displays/) are 2x what's displayed.
```

Comment:
```suggestion
The pre-push workflow will validate image size and fail if there are images bigger than 400kb in size. It will also warn you about using PNG images as they are lossless and occupy more space. For a blog post, JPEGs or WebP images are better suited. So do consider using those formats. And ideally, images should be under 1800px in width. The blog content is rendered in a 900px width container and [retina images](https://www.sleeplessmedia.com/2018/12/14/optimizing-website-images-and-graphics-for-retina-displays/) are 2x what's displayed.
```